### DISCUSSION TITLE
Search doesn't support emoji

### DISCUSSION THREAD

**Discussion by idbrii** [community/community#6539] [Other Feature Feedback, Questions, & Ideas] [answered]
As described in #6093, vim-win32-installer uses 🔒 to unambiguously tag signed releases (since the word "unsigned" contains the word "signed", the emoji makes it easier to search).

Searching [gives no results for 🔒](https://github.com/vim/vim-win32-installer/releases?q=%F0%9F%94%92&expanded=true), but [v8.2.3536](https://github.com/vim/vim-win32-installer/releases/tag/v8.2.3536) contains that emoji and it's the second [result for "signed"](https://github.com/vim/vim-win32-installer/releases?q=signed&expanded=true).

**Comment by MylesBorins** [comment, marked as answer]
Thanks for letting us know. I've updated engineering about this and we'll see what we can do
